desk/tests/test_console_commands.py

```diff
@@ -74,15 +152,24 @@ def test_single_fit_options(tmpdir):
     )
 
 
-def test_single_sed(tmpdir):
+def test_single_sed(tmpdir, image_similarity):
     # test for single SED figure
-    console_commands.sed()
+    create_sample_data(tmpdir, 1)
+    create_sample_data(tmpdir, 2)
+    create_sample_data(tmpdir, 3)
+    example_fitting_results = create_fitting_results(tmpdir)
+    console_commands.sed(
+        source_path=tmpdir.strpath,
+        source_filename="fitting_results.csv",
+        dest_path=tmpdir.strpath,
+        save_name="test_single_sed.png",
+    )
 
 
 def test_multiple_fit(tmpdir):
     # test for multiple SED fit with SED figure
     create_sample_data(tmpdir, 1)
     create_sample_data(tmpdir, 2)
     create_sample_data(tmpdir, 3)
-    console_commands.fit(source=str(tmpdir))
-    console_commands.sed()
+    example_fitting_results = create_fitting_results(tmpdir)
+    console_commands.sed_indiv
```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/manual/s-goldman/Dusty-Evolved-Star-Kit/pullRequest?prid=5896643)